Makefile.COMMON

```diff
@@ -44,11 +44,7 @@ SRC    ?= $(shell find . -type f -name "*.go" ! -path "./.build/*")
 GOOS   ?= $(shell uname | tr A-Z a-z)
 GOARCH ?= $(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m)))
 
-ifeq ($(GOOS),darwin)
-	RELEASE_SUFFIX ?= -osx$(shell sw_vers -productVersion)
-endif
-
-GO_VERSION ?= 1.4.2
+GO_VERSION ?= 1.5.1
 GOURL      ?= https://golang.org/dl
 GOPKG      ?= go$(GO_VERSION).$(GOOS)-$(GOARCH)$(RELEASE_SUFFIX).tar.gz
```

Comment:
Doh. Well it gives an easy way to use 1.4 in projects ;). I won't have
access to a computer the next two days.
On Sep 11, 2015 7:09 PM, "Julius Volz" <notifications@github.com> wrote:

> In Makefile.COMMON
> <https://github.com/prometheus/utils/pull/15#discussion_r39327319>:
>
> > @@ -44,11 +44,7 @@ SRC    ?= $(shell find . -type f -name "*.go" ! -path "./.build/*")
> >  GOOS   ?= $(shell uname | tr A-Z a-z)
> >  GOARCH ?= $(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m)))
> >
> > -ifeq ($(GOOS),darwin)
> > -	RELEASE_SUFFIX ?= -osx$(shell sw_vers -productVersion)
> > -endif
> > -
> > -GO_VERSION ?= 1.4.2
> > +GO_VERSION ?= 1.5.1
> >  GOURL      ?= https://golang.org/dl
> >  GOPKG      ?= go$(GO_VERSION).$(GOOS)-$(GOARCH)$(RELEASE_SUFFIX).tar.gz
>
> Also remove the RELEASE_SUFFIX here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/utils/pull/15/files#r39327319>.
>